modules/core/src/main/scala/org/scalasteward/core/edit/EditAlg.scala

```diff
@@ -58,9 +58,7 @@ final class EditAlg[F[_]](
   }
 
   def applyScalafixMigrations(repo: Repo, update: Update): F[Unit] =
-    migrationAlg.findMigrations(update) >>= { migrationsList =>
-      Nel.fromList(migrationsList).fold(F.unit) { migrations =>
-        logger.info(s"Applying migrations: $migrations") >> sbtAlg.runMigrations(repo, migrations)
-      }
+    Nel.fromList(migrationAlg.findMigrations(update)).fold(F.unit) { migrations =>
```

Comment:
it's traverse!

Reply:
It always is. :-) Thanks, I'll change it.